doc/spec/Types.md

```diff
@@ -155,12 +161,59 @@ var x = undefined;      // Same as x: any = undefined
 var e: Undefined;       // Error, can't reference Undefined type
 ```
 
+#### Literal types { #literal-types }
+
+TypeScript supports string, number and boolean literal types, but not regular expression literal types.
+
+Literal types use the syntax of their respective literal values and have the identities of their respective literal values.
+For example, `10` and `1e1` are the same type, even though the two are written differently.
+
+Similarly, literal types are subclasses of their respective literal values' types:

```

Comment:
subtypes of their respective **base primitive type**:

Reply:
do you want this bolded in the text too?